src/main/java/org/folio/circulation/domain/notice/TemplateContextUtil.java

```diff
@@ -414,13 +417,20 @@ public UserContext withAddressProperties(JsonObject address) {
           .with(UserContext.CITY, address.getString("city", null))
           .with(UserContext.REGION, address.getString("region", null))
           .with(UserContext.POSTAL_CODE, address.getString("postalCode", null))
-          .with(UserContext.COUNTRY_ID, address.getString("countryId", null))
+          .with(UserContext.COUNTRY_ID, getCountryNameByCountryId(address.getString(COUNTRY_ID, null)))
           .with(UserContext.ADDRESS_TYPE_NAME, address.getString("addressTypeName", null));
       } else {
         return this;
       }
     }
 
+    private String getCountryNameByCountryId(String countryId) {
+      if(StringUtils.isNotEmpty(countryId)){
```

Comment:
A tiny thing, but I believe we're styling `if`s with the space more often in mod-circulation: `if (`